docroot/themes/humsci/humsci_basic/src/js/shared/animation/page-scroll.js

```diff
@@ -4,7 +4,7 @@ let scroll = window.requestAnimationFrame || function(callback) {
 };
 
 // The classes of items we want to add animations to
-let showAnimation = document.querySelectorAll('.hb-hero-overlay, .hb-hero-overlay__text, .hb-hero-overlay__image-wrapper, .field-hs-hero-image');
+let showAnimation = document.querySelectorAll('.hb-hero-overlay, .hb-hero-overlay__text, .hb-hero-overlay__image-wrapper, .field-hs-hero-image, .hb-gradient-hero, .hb-gradient-hero__text, .hb-gradient-hero__image-wrapper, .field-hs-gradient-hero-image');
```

Comment:
Now might be a good time to refactor these classes into an array:

```js
const classesToAnimate = [
  '.hb-hero-overlay',
  '.hb-hero-overlay__text',
  '.hb-hero-overlay__image-wrapper',
  '.field-hs-hero-image',
  '.hb-gradient-hero',
  '.hb-gradient-hero__text',
  '.hb-gradient-hero__image-wrapper',
  '.field-hs-gradient-hero-image'
];

let showAnimation = document.querySelectorAll(classesToAnimate);
```

Reply:
Yes!!!!! :tada: Excellent suggestion!